### ISSUE TITLE
Only downloads back to a few hours ago.

### ISSUE THREAD

**Issue by Beniled** [nayrk/Blink#21]
How do I have it download all videos. I appears to stop at a couple of hours ago. Is there something I need to do?

**Comment by stevekenney318** [comment]
Not sure if this is your issue or not, but the program will only run for 24 hrs successfully .  This is an Amazon/Blink thing. If it's run for 24 hrs, you need to close it and restart it again with a new code. I modified mine to add a timestamp and show it every time it runs. That way I can tell if it has 'expired' or not. I would suggest they add that to the program. See the picture attached. I also changed mine to run every 3 hours ... gives me a chance to clear some captures from the app if I need to. 
![20220506_115426_002024](https://user-images.githubusercontent.com/53465149/167169270-79359f19-97ca-4246-ab7f-6fc13767e53b.jpg)

I'm hoping naryk or other programmers will see your issue and comment ... I have requested assistance with downloading thumbnails - https://github.com/nayrk/Blink/issues/19  .  No response. Matter of fact yours is the 5th issue with no response ... Last activity by nayrk was in March of 2021... 

Just curious, are you getting your thumbnails downloaded ? Here's what I get... 
![20220506_120527_002025](https://user-images.githubusercontent.com/53465149/167170471-34f817aa-8a99-4eed-82e5-afa442ceb920.jpg)

Regards, Steve

**Comment by Beniled** [comment]
No what I mean is, I just started running it yesterday at around 3pm. I have several weeks worth of videos on 4 different cameras but the earliest video it would download is timecoded at 1:28pm yesterday. It won't download anything earlier or on previous days. If I empty the folder it only downloads to around the timestamp of the last video it downloaded. I want to download all previous videos that are currently in storage.

**Comment by stevekenney318** [comment]
> No what I mean is, I just started running it yesterday at around 3pm. I have several weeks worth of videos on 4 different cameras but the earliest video it would download is timecoded at 1:28pm yesterday. It won't download anything earlier or on previous days. If I empty the folder it only downloads to around the timestamp of the last video it downloaded. I want to download all previous videos that are currently in storage.

Oh ok, now I get it.  I'm sorry, I don't know what would cause that... I've been using it for a while, and I don't remember anything like that when I started... Sorry.

**Comment by imac211** [comment]
I too would like to figure out how to download what is in storage as well...I can't quite figure out where it is grabbing videos marked "new"

**Comment by stevekenney318** [comment]
> I too would like to figure out how to download what is in storage as well...I can't quite figure out where it is grabbing videos marked "new"

My thought is it's only grabbing what's currently showing on your app. I have no idea what the location is. 

I'm still having issues downloading new thumbnails. Do you have that issue ? See -> https://github.com/nayrk/Blink/issues/19

**Comment by imac211** [comment]
I do not have that problem.  I'm still able to pull the newest videos (it may take awhile sometimes), but if I leave it up, it still works.  Ignore what I previously said about making sure to use latest version, as I do not see any other versions available.

**Comment by stevekenney318** [comment]
ok.

**Comment by jg24001** [comment]
Yes I have the same thumbnail problem, just started a few months ago. I don't care about thumbnails so much so I ignored it. Just found out today that it isn't downloading doorbell cameras. Must be on a different server than the regular cameras?

**Comment by stevekenney318** [comment]
> Yes I have the same thumbnail problem, just started a few months ago. I don't care about thumbnails so much so I ignored it. Just found out today that it isn't downloading doorbell cameras. Must be on a different server than the regular cameras?

Sorry to hear it isn't downloading the doorbell camera videos... I do care about the thumbnails, and I would hope that the original author or one of those that have made modifications to the program would be able to help out.. Tt seems like there is a variable issue in creating the downloaded name and that the files aren't in the location they thought they were in.... Hope someone can help us both out.

**Comment by CBR9612** [comment]
Can I ask, how old are the videos you have in storage? There is a line in the code (hardcoded) to ignore videos over some number of years old.  Can't imagine that is the issue, but thought I would ask.

**Comment by jg24001** [comment]
I'm not having the few hours old problem, I'm only having the doorbell not
downloading (must be on a different server somehow?) And the old thumbnails
not updating, see the error screenshots others have posted. I don't care
about the thumbnails though, the person who raised that issue does.

Love the script thank you!!!!
